src/Features/CSharp/Portable/EditAndContinue/CSharpEditAndContinueAnalyzer.cs

```diff
@@ -21,8 +23,6 @@
 using Microsoft.CodeAnalysis.Text;
 using Roslyn.Utilities;
 
-#nullable enable
```

Comment:
Why remove nullable? Do we enable it now for the whole project?

Reply:
I didn't remove it. I fixed the misplaced directive.

Reply:
I see.